src/core/public/chrome/ui/header/collapsible_nav_groups.tsx

```diff
@@ -58,7 +58,11 @@ export function NavGroups({
 
     return {
       id: `${link.id}-${link.title}`,
-      name: <EuiText size="xs">{link.title}</EuiText>,
+      name: (
+        <EuiText size="xs" style={{ fontSize: '14px', lineHeight: '16px', fontWeight: 400 }}>
```

Comment:
Shall we not use custom font size?

Reply:
I tried several combination with EuiText size + <h{n}> and I had to add one more customized inline-style `fontWeight: 400` for making it look good.